test/e2e/framework/util.go

```diff
@@ -442,14 +442,14 @@ func NodeOSDistroIs(supportedNodeOsDistros ...string) bool {
 }
 
 // ProxyMode returns a proxyMode of a kube-proxy.
-func ProxyMode(f *Framework) (string, error) {
+func ProxyMode(f *Framework, hostNetwork bool) (string, error) {
```

Comment:
I was reading the comments from previous thread (https://github.com/kubernetes/kubernetes/pull/69525#discussion_r230224108). Isn't that still true?

Reply:
@bclau This function must be able to access kube-proxy running on the node, so running in non-hostNetwork doesn't work at all. You didn't see a problem because apparently `RunHostCmd` doesn't return an error when the `curl` command times out or gets a connection refused. So basically the test was then going on assuming it's not in user proxy mode. How is kube-proxy done in Windows, given there is no iptables? Does it run the user space proxy? If so then an "if windows, return userspace" might be the right solution for this function.

Alternately, the issue is that Windows doesn't support hostNetwork, and we need to talk to kube-proxy in the host network namespace in order to determine the proxy mode. So how do we get this info without hostNetwork? Does `kube-proxy` only make status available on localhost, or could we get it by passing `status.hostIP` via the downward API? If we can do that, then the pod can get this info using that IP instead of `localhost`.

Reply:
Will remove this from the PR for the time being. Didn't notice that the test actually skips if it doesn't have 2 ``Ready`` nodes, so I've missed this.

Reply:
Multiple nodes are required now.

No conformance tests should have Skips in them. If they do, they cannot be promoted.